src/transformers/models/swinv2/modeling_swinv2.py

```diff
@@ -683,7 +683,9 @@ def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
 
 
 class Swinv2Layer(nn.Module):
-    def __init__(self, config, dim, input_resolution, num_heads, shift_size=0, pretrained_window_size=0):
+    def __init__(
+        self, config, dim, input_resolution, num_heads, drop_path_rate=0.0, shift_size=0, pretrained_window_size=0
+    ):
```

Comment:
As mentioned in the other linked PR I think it'd be nice to refactor the init of all of these layers - the reason is that we're trying to have not too many args, and here we're adding yet another one. `input_resolution`, `num_heads` etc can all be derived from `config` and `layer_idx` (albeit through `grid_size` for `input_resolution`). We should modify this in a follow-up PR, or in this one if you feel up to it!

Reply:
happy to do it follow-up pr

Reply:
sounds good then!